AngularJS/app.js

```diff
@@ -0,0 +1,214 @@
+angular.module("reportApp", ['main']);
+var app = angular.module("main", []);
+
+angular.module('reportApp')
+    .service('reportService', ReportService)
```

Comment:
Move service and components to different files.
Update folder structure for the frontend that divides different module in folders